crmsh/bootstrap.py

```diff
@@ -228,13 +231,49 @@ def wait_for_cluster():
     status_long("Waiting for cluster")
     while True:
         _rc, out, _err = utils.get_stdout_stderr("crm_mon -1")
-        if "online" in out.lower():
+        if is_online(out):
             break
         status_progress()
-        sleep(5)
+        sleep(2)
     status_done()
 
 
+def get_peer_hostname():
+    """
+    In join process, cluster_node might be an IP or hostname
+    Return hostname anyway
+    """
+    peer_node = None
+    if _context.cluster_node:
+        if utils.valid_ip_addr(_context.cluster_node):
+            peer_node = utils.get_stdout("ssh {} hostname".format(_context.cluster_node))[1]
```

Comment:
What would happen here in this scenario:
- The IP address is valid, but it's not available for SSH or it doesn't even exist.

Reply:
Changed, use `utils.get_stdout_stderr` instead, and check the return code